apps/minifront/src/components/shared/input-block.tsx

```diff
@@ -14,20 +15,22 @@ export const InputBlock = ({ label, className, validations, value, children }: I
   const vResult = typeof value === 'string' ? validationResult(value, validations) : undefined;
 
   return (
-    <div
-      className={cn(
-        'bg-background px-4 pt-3 pb-4 rounded-lg border flex flex-col gap-1',
-        vResult?.type === 'error' && 'border-red-400',
-        vResult?.type === 'warn' && 'border-yellow-300',
-        className,
-      )}
-    >
-      <div className='flex items-center gap-2'>
-        <p className='text-base font-bold'>{label}</p>
-        {vResult ? <div className={cn('italic', 'text-red-400')}>{vResult.issue}</div> : null}
-      </div>
+    <Box>
```

Comment:
No change here other than removing the `Box`-specific classes from the div, and wrapping everything in a `<Box />`